### DISCUSSION TITLE
Vscode Unable To Give Props Autocompletion

### DISCUSSION THREAD

**Discussion by mockingjet** [vuejs/language-tools#2382] [Q&A] [answered]
I made a shared component library, and build it with typescript then import it to my current project.

![image](https://user-images.githubusercontent.com/32839503/218652753-2ba4ce4d-c440-4abe-8a93-7971b009271d.png)

When my mouse hovered on the component name, it did show the component props.

![image](https://user-images.githubusercontent.com/32839503/218652932-ac53a215-2b22-44b2-bdaa-ff59c4fd8b4b.png)

But when I was typing `disabled`, vscode didn't show autocomplete options related to the props.

Which part did I miss?

**Comment by johnsoncodehk** [comment, marked as answer]
This is a bug, fixed by https://github.com/vuejs/language-tools/commit/e3dc9b5964f219ae1b6c11b1a3d10e539b28f8ae.
